src/api/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 add_library(api OBJECT string.cpp exception.cpp name.cpp options.cpp univ.cpp
   expr.cpp decl.cpp env.cpp ios.cpp module.cpp type_checker.cpp inductive.cpp
   parser.cpp)
-
+if(EMSCRIPTEN)
+add_dependencies(api gmp)
```

Comment:
Do you know why this is only needed with emscripten?  And why it wasn't necessary before?

Reply:
This is only needed for emscripten because only emscripten has to build the ExternalProject `gmp` whereas the non-emscripten builds are required to have gmp already installed. I'm not entirely sure why it wasn't necessary before. It was only after I split up the `util` target into `util_shared` and `util_path(_browser)` that I started seeing `make` try to build some of these other targets without building `gmp` first, so I added these explicit dependencies to address that.

Reply:
Ah, now I remember: this didn't work before either.  I just always run `emmake make gmp` first.